code/game/sound.dm

```diff
@@ -65,7 +65,7 @@ GLOBAL_LIST_INIT(bubble_small_sound,list('sound/machines/tanksmallbubble1.ogg','
 GLOBAL_LIST_INIT(fleshtear_sound, list('sound/effects/organic/flesh_tear_1.ogg','sound/effects/organic/flesh_tear_2.ogg','sound/effects/organic/flesh_tear_3.ogg',))
 
 
-/proc/playsound(var/atom/source, soundin, vol as num, vary, extrarange as num, falloff, var/is_global, var/frequency, var/is_ambiance = 0)
+/datum/proc/playsound(var/atom/source, soundin, vol as num, vary, extrarange as num, falloff, var/is_global, var/frequency, var/is_ambiance = 0)
```

Comment:
this is a huge change which seems questionable
Datums don't exist in physical space, so they can't emit sounds in a meaningful way. They could be responsible for emitting sounds, but i don't believe there's any need to track that outside of looped audio

I'd be surprised if this doesn't break something. If there's a need for it please explain